client/app/scripts/components/node-details/node-details-health.js

```diff
@@ -20,7 +21,12 @@ export default class NodeDetailsHealth extends React.Component {
   }
 
   render() {
-    const metrics = this.props.metrics || [];
+    const {
+      metrics = makeList(),
```

Comment:
I'd be worried that you're mixing immutablejs and JS array functions further down. E.g., immutablejs prefers `.size` to check list length.